source/tomopy/recon/algorithm.py

```diff
@@ -392,10 +415,28 @@ def _dist_recon(tomo, center, recon, algorithm, args, kwargs, ncore, nchunk):
             # run in this thread (useful for debugging)
             algorithm(tomo[slc], center[slc], recon[slc], *args, **kwargs)
     else:
-        # execute recon on ncore threads
-        with cf.ThreadPoolExecutor(ncore) as e:
-            for slc in use_slcs:
-                e.submit(algorithm, tomo[slc], center[slc], recon[slc], *args, **kwargs)
+        # execute recon on ncore processes. Accelerated methods have internal thread-pool
+        # and NVIDIA NPP library does not support simulatenously leveraging multiple devices
+        # within the same process
+        if "accelerated" in kwargs and kwargs["accelerated"]:
+            futures = []
+            with cf.ProcessPoolExecutor(ncore) as e:
+                for idx, slc in enumerate(use_slcs):
+                    futures.append(
+                        e.submit(_run_accel_algorithm, idx, algorithm,
+                                 tomo[slc], center[slc], recon[slc], *args,
+                                 **kwargs))
+
+            for f, slc in zip(futures, use_slcs):
+                if f.exception() is not None:
+                    raise f.exception()
+                recon[slc] = f.result()
+        else:
+            # execute recon on ncore threads
+            with cf.ThreadPoolExecutor(ncore) as e:
+                for slc in use_slcs:
+                    e.submit(algorithm, tomo[slc], tomo[slc], center[slc],
+                             recon[slc], *args, **kwargs)
```

Comment:
```suggestion
                    e.submit(algorithm, tomo[slc], center[slc],
                             recon[slc], *args, **kwargs)
```
This argument is should not be duplicated.

Reply:
Haha, yep, that'll do it.